cmd/bpf2go/gen/output_test.go

```diff
@@ -12,58 +13,6 @@ import (
 	"github.com/cilium/ebpf/cmd/bpf2go/internal"
 )
 
-func TestOrderTypes(t *testing.T) {
```

Comment:
Basically same question as above. Do we have another way of ensuring stability in the generated output? Was this always unnecessary?

Reply:
When we `range` over a map in `text/template`, items are first ordered by key.